doc/cmake.rst

```diff
@@ -86,20 +85,19 @@ hardware is unavailable.
 Shared Library Naming and Versioning
 ####################################
 
-Both OpenCL and Vulkan APIs are designed to work through an ICD loader, a
-shared library named ``OpenCL`` and ``Vulkan`` respectively in the form
+Both OpenCL APIs are designed to work through an ICD loader, a
```

Comment:
This sentence is no longer right, "Both" referred to "OpenCL and Vulkan APIs", there is no "Both OpenCL APIs".